spark/v3.0/spark-extensions/src/main/scala/org/apache/spark/sql/execution/datasources/v2/DynamicFileFilterExec.scala

```diff
@@ -41,6 +43,11 @@ abstract class DynamicFileFilterExecBase(
   @transient
   override lazy val references: AttributeSet = AttributeSet(fileFilterExec.output)
 
+  override lazy val metrics = Map(
+    "totalFiles" -> SQLMetrics.createMetric(sparkContext, "total number of files"),
```

Comment:
Can you make these display names a bit shorter and capitalize them? I think they should be "Candidate files" and "Matching files". Those are descriptive, but short.

Reply:
I modified the strings to "candidate files" and "matching files".

But I probably won't modify to capitalize. This is for consistency with other metric formats.

![metric_formats](https://user-images.githubusercontent.com/67534059/148897414-22cc321a-fcd9-4662-8f1a-683ed590e693.jpg)